packages/model-viewer/src/features/animation.ts

```diff
@@ -41,6 +41,7 @@ export declare interface AnimationInterface {
   readonly paused: boolean;
   readonly duration: number;
   currentTime: number;
+  currentTimeScale: number;
```

Comment:
maybe just `timeScale`?

Reply:
I was aware of the `currentTime`, but you're right.